index.bs

```diff
@@ -3303,58 +3358,72 @@ To <dfn>get the realm info</dfn> given |environment settings|:
 
 1. Let |realm id| be the [=realm id=] for |realm|.
 
+1. Let |origin| be the [=serialization of an origin=] given |environment settings|'s |origin|.
+
 1. Run the steps under the first matching condition:
 
   <dl>
     <dt>The [=Realm/global object=] specified by |environment settings| is a [=Window=] object
```

Comment:
Looks like the question is out of the scope of the PR, but still I ask: what the check if `smth is a Window object` will look like? Should it be specified based on some internal fields or something like that?